.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -1,16 +1,21 @@
 name: update-and-publish-cromwell-helm-chart
 on:
-  # push:
-  #   branches:
-  #     - 'main'
-  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
 
 jobs:
   update-and-publish:
     if: ( !startsWith(github.event.head_commit.message, 'Auto update cromwell-helm chart version') )
     name: Update and publish new cromwell-helm chart
     runs-on: ubuntu-latest
     steps:
+      - name: Ensure that a Jira ID is present in the commit message
```

Comment:
TOL:

- Can we have a github action that checks that PR titles have jira tags before allowing us to commit?
- Can we validate/change our repo settings to make sure the PR title becomes (at least part of) the commit message?